tests/rules/test_anchors.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
```

Comment:
This line shouldn't be needed now that yamllint only supports Python 3. Can you remove it (along with the one from `yamllint/rules/anchors.py`)?